ksqldb-engine/src/main/java/io/confluent/ksql/function/UdfLoaderUtil.java

```diff
@@ -22,10 +22,11 @@
 public final class UdfLoaderUtil {
   private UdfLoaderUtil() {}
 
-  public static FunctionRegistry load(final MutableFunctionRegistry functionRegistry) {
+  public static FunctionRegistry load(final MutableFunctionRegistry functionRegistry,
```

Comment:
Unrelated to your change, but noticed this class is in the production code, yet is clearly a test class.  Can we move it into the test directory please?

the functional testing module already depends on the engine's test jar.